Makefile

```diff
@@ -89,10 +94,10 @@ deploy: # Deploy/redeploy previously built REANA cluster.
 	docker images | grep '<none>' | awk '{print $$3;}' | xargs -r docker rmi && \
 	reana-cluster -f ${PWD}/../reana-cluster/reana_cluster/configurations/reana-cluster-latest.yaml init --traefik
 
-example: # Run a simple "hello world" analysis example.
+example: # Run a REANA example. By default all REANA examples are executed.
 	source ${HOME}/.virtualenvs/${VENV_NAME}/bin/activate && \
 	eval $$(reana-dev setup-environment) && \
-	reana-dev run-example -c r-d-helloworld -s ${TIME_SLEEP}
+	reana-dev run-example -c $${EXAMPLE:-DEMO} -s ${TIME_SLEEP}
```

Comment:
Maybe then call it `DEMO=r-d-helloworld make example` so that we use DEMO consistently everywhere?